windows/README.md

```diff
@@ -0,0 +1,34 @@
+# Windows Resources
+
+Windows agents can be wrapped with [winsw][] to run as Windows services.
```

Comment:
Missing URL?

Reply:
This is actually the markdown syntax for a footnote style link. the empty second bracket indicates that the link text is also the footnote key. The URL is on line 33.